### PR TITLE
GL-925: Fix GL check eligibility start links.

### PR DESCRIPTION
**Pull request by alexdesi**
### Jira link
https://transformuk.atlassian.net/browse/GL-925

### What?
Fix GL check eligibility start links.
The XI of the GL link does not exist.

### Why?
To fix broken links

### CODE REVIEW

File: app/helpers/green_lanes_helper.rb [hunk 1]
```diff
@@ -63,6 +63,10 @@ def yes_no_not_sure_options
     [OpenStruct.new(id: 'yes', name: 'Yes'), OpenStruct.new(id: 'no', name: 'No'), OpenStruct.new(id: 'not_sure', name: 'Not sure')]
   end
 
+  def green_lanes_eligibility_start_path
```
Comment (shjohnson) [inline app/helpers/green_lanes_helper.rb:66]: Just thought, do we start changing the method name now? something like;

`def spimm_eligibility_start_path`

**Review by shjohnson — COMMENTED** [review]
I think also we need to fix:

`app/views/pages/_howto_sidebar_links.html.erb`

File: app/helpers/green_lanes_helper.rb (hunk 1 above)
Comment (alexdesi) [reply app/helpers/green_lanes_helper.rb:66]: good point, but it's probably better to create a ticket to rename all of those,
there are 267 occurrences of 'green lanes' !
so I'd avoid to create mixed notation

**Review by shjohnson — APPROVED** [review]
💚

**Pull request merged**